### PR TITLE
Meshrdf klortho danizen

### PR DESCRIPTION
**Pull request by Klortho**

### CODE REVIEW

File: xslt/mesh-rdf-prefixes.ent.template [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!ENTITY mesh "http://id.nlm.nih.gov/mesh/">
+<!ENTITY mesh "http://id.nlm.nih.gov/MESHRDF_PATH/">
 <!ENTITY meshv "http://id.nlm.nih.gov/mesh/vocab#">
```
Comment (Klortho) [inline xslt/mesh-rdf-prefixes.ent.template]: I think both `mesh:` and `meshv:` URIs will either not have the year in them ("current") or have the year in them. You only have `mesh:` parameterized here.

**Comment by danizen** [comment]
@Klortho, I did have that question.   I noticed while writing my load script that meshrdf contains `vocabulary.owl` but `/usr/nlm/mesh/data` contains `vocabulary.ttl` - what is the conversion process?

**Comment by Klortho** [comment]
Got me!  If there's any conversion process, I don't know it. Somebody must have done that manually (saved it in turtle format from Protege).  But anyway, the format of that file doesn't effect the form of the URIs.

**Comment by Klortho** [comment]
We also need to generate "current" URIs, which won't have any year.  Do you mind if I try to fix this and push changes up here?

**Comment by danizen** [comment]
My plan was to use two conversions to do this, in sequence

```bash
MESHRDF_YEAR=2015 mesh-xml2rdf.sh
MESHRDF_YEAR=2015 MESHRDF_PREFIX=mesh/current mesh-xml2rdf.sh
```

I've tested this and it works:

```bash
davisda4@dvlbmeshrdf01: /usr/nlm/mesh/opt/danizen/meshrdf$ ls out
mesh2014-dups.nt  mesh2015-dups.nt  mesh2015.nt  mesh2015.nt.gz  meshcurrent-dups.nt  meshcurrent.nt  meshcurrent.nt.gz
````

**Comment by Klortho** [comment]
Okay, sounds good.  Only one thing, I think the second line should be

    MESHRDF_YEAR=2015 MESHRDF_PREFIX=mesh mesh-xml2rdf.sh

**Comment by danizen** [comment]
> Hang on ... then we need a naming convention for "2015 qua current" vs. "2015 qua 2015"

You mean in terms of urlregexp.xml?   Or, something else?

**Comment by Klortho** [comment]
See [this](https://github.com/HHS/meshrdf/pull/25).  Let me know if you don't like it.

**Comment by flamingofugang** [comment]
Sorry to catch up late, I did generated both vocabulary.owl and vocabulary.ttl. We can get both RDF format from protege editor. vocaburlary.ttl was intended to be loaded into virtuoso. I encountered some syntax error when I loaded vocabulary.owl into virtuoso. The xml syntax between protege and virtuoso are not compatible somehow, I have reported to virtuoso community earlier.

**Pull request merged**